modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -1,5 +1,37 @@
 [id='con-relnotes-fixed-issues_{context}']
-= Fixed issues
+= Fixed issues in {product} {product-version} and 1.2.2
+
+== Fixed issues in {product} 1.2.2
+
+This section lists fixed issues with {product} 1.2.2:
+
+Dynamic plugins fail to load if the file name in ConfigMap is invalid::
++
+--
+Previously, an invalid dynamic plugin file name in ConfigMap prevented the creation of a Backstage CR. With this release, the Backstage CR can be created even with an invalid plugin file name, and a corresponding log message will be displayed if the {product-short} build fails.
+--
+
+Plugin ID is missing in the permission table in RBAC Administration UI::
++
+--
+Previously, the plugin ID was missing from the permission table in the *Administration* -> *RBAC*. With this release, updates to both the RBAC backend and frontend plugins now ensure the plugin ID is displayed in the table.
+--
+
+Deprecation warning about `backend.auth.keys` in application log::
++
+--
+Previously, {product-short} build logs showed the following deprecation warning:
+
+`DEPRECATION WARNING: The backend.auth.keys config has been replaced by backend.auth.externalAccess. See https://backstage.io/docs/auth/service-to-service-auth`
+
+With this release, if your deployment uses the default configuration, the deprecation warning will no longer appear in the application logs.
```

Comment:
because we updated to the latest? Because we're hiding the warning? I feel like some context is missing here.

Reply:
can you point at the code where this change occurred? JIRA?

Reply:
Done, Moved this to known issues.